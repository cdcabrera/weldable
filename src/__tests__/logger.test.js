const logger = require('../logger');

describe('Logger', () => {
  it('should return specific properties', () => {
    expect(logger).toMatchSnapshot('specific properties');
  });
});
