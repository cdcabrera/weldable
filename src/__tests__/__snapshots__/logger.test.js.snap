// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Logger should allow multiple console methods: error 1`] = `
[
  [
    "[31merror message[0m",
  ],
]
`;

exports[`Logger should allow multiple console methods: info 1`] = `
[
  [
    "[34minfo message[0m",
  ],
]
`;

exports[`Logger should allow multiple console methods: warn 1`] = `
[
  [
    "[33mwarn message[0m",
  ],
]
`;

exports[`Logger should return specific properties: specific properties 1`] = `
{
  "color": {
    "BLACK": "[30m",
    "BLUE": "[34m",
    "CYAN": "[36m",
    "GREEN": "[32m",
    "GREY": "[90m",
    "MAGENTA": "[35m",
    "NOCOLOR": "[0m",
    "RED": "[31m",
    "WHITE": "[37m",
    "YELLOW": "[33m",
  },
  "consoleMessage": {
    "error": [Function],
    "info": [Function],
    "log": [Function],
    "success": [Function],
    "warn": [Function],
  },
}
`;
